### ISSUE TITLE
Interactive timeline facet

### ISSUE THREAD

**Issue by bkmgit** [OpenRefine/OpenRefine#3229]
It would be helpful if one could make the timeline facet interactive so that when there are many entries, one can query for exact values from the display

### Proposed solution
Make the timeline facet interactive, so that when mouse cursor is moved over an entry, the value of that entry appears.
### Alternatives considered
Have not considered alternatives.
### Additional context
![DeepinScreenshot_select-area_20200929103914](https://user-images.githubusercontent.com/1679477/94528183-b30bf580-0240-11eb-8b5b-b02ac5a99534.png)

**Comment by wetneb** [comment]
> Make the timeline facet interactive, so that when mouse cursor is moved over an entry, the value of that entry appears.

It is not clear to me what you are proposing here. What is an "entry" here? For me, the timeline facet does not have entries like the text facet, but a list of histogram bars.

**Comment by bkmgit** [comment]
> It is not clear to me what you are proposing here. What is an "entry" 
> here? For me, the timeline facet does not have entries like the text 
> facet, but a list of histogram bars.
> 
The value of the histogram bar should appear when mouse hovers over it.

**Comment by wetneb** [comment]
Fair enough, thanks for the clarification! I agree that would be useful.

**Comment by tfmorris** [comment]
I think there are probably three values of interest:
- datetime for lower bound of histogram bucket
- datetime for upper bound of histogram bucket
- count of rows in the bucket

It might be useful to elide information of minimal value e.g. hh:mm of 00:00 or end date which is exactly 24 hours later ie buckets are a single day.

**Comment by thadguidry** [comment]
For what it's worth, long ago I asked David to provide a more granular timeline facet that I could control more flexibly such as in Informatica and Tableau's tooling.  Here's a D3.js example of that, where the user can control the granularity via a vertical zoom slider, while also still being able to pan left to right on the time series slider:

https://rawgit.com/patternfly/patternfly-timeline/master-dist/demo/
